docs/reference/docs/multi-get.asciidoc

```diff
@@ -85,16 +93,18 @@ will give you back only the same document twice:
 
 [source,js]
 --------------------------------------------------
-curl 'localhost:9200/test/_mget' -d '{
+GET test/_mget
+{
     "ids" : ["1", "1"]
-}'
+}
 --------------------------------------------------
+// CONSOLE
 
 You need in that case to explicitly set the `_type`:
 
 [source,js]
 --------------------------------------------------
-GET /test/_mget/
+GET test/_mget/
```

Comment:
I tend to leave the `/` on the front but not on the end. Either way is fine though.